eslint.config.mjs

```diff
@@ -53,7 +53,6 @@ export default tseslint.config(
 			"@typescript-eslint/no-deprecated": "off",
 			"@typescript-eslint/no-dynamic-delete": "off",
 			"n/no-missing-import": "off",
-			"n/no-unsupported-features/node-builtins": "off",
```

Comment:
I remember this rule causing us grief when we had node 18 supported.  But removing the disable doesn't introduce new violations, so 🤷‍♂️

Reply:
Blagh yeah this rule and the reporting on experimental `parseArgs`! Very excited to remove its suppressions all around. 🔥